### PR TITLE
Update

### PR DESCRIPTION
**Pull request by blackjackal982**
Changes done in insert function

### CODE REVIEW

File: testcode.c [hunk 1]
```diff
@@ -56,22 +79,59 @@ void display(list_t* list)
...
+
+int getFloatSize()
+{
+    return sizeof(float);
+}
+
+int getDoubleSize()
+{
+    return sizeof(double);
+}
```
Comment (har130798) [inline testcode.c]: write functions for get_char_size and get_string_size

File: testcode.c [hunk 2]
```diff
@@ -1,50 +1,73 @@
...
-        nxt->next->next=NULL;
+        for(temp = list->head ; temp->next != NULL ; temp = temp->next);
+        current = temp->next;
+
     }
+
+    current = create_node();
+    current->data = malloc(getSize());
+
+    memcpy(current->data,value,getSize());
```
Comment (souravabhijith) [inline testcode.c]: Line 52 + 54 can be moved into create_node itself

File: testcode.c [hunk 3]
```diff
@@ -1,50 +1,73 @@
...
+
+    list->size++;
+    current->next = NULL;
+
+    if(flag==1)
+    {
+        list->head = current;
+    }
+    else
```
Comment (souravabhijith) [inline testcode.c]: Node can be created before first if and else itself, and overall this function can only have one if -else, think properly

Comment (blackjackal982) [reply testcode.c]: Ok

**Pull request closed without merging**